tests/align/align_test.py

```diff
@@ -0,0 +1,170 @@
+from align_utils import TensorAlignmentData, align_tensors
+import os
+import sys
+from typing import Callable
+
+sys.path.append("./align/")
+
+
+BASE_DIR = "tests/align/out"
+param_weight_op = {'conv2d', 'embedding', 'view_embedding'}
+param_bias_op = {'conv2d'}
+no_grad_op = {"getitem"}
+
+
+def prepend_dirname_fn(dirname: str) -> Callable[[str], str]:
+    def f(filename):
+        return os.path.join(dirname, filename)
+    return f
+
+
+def test_embedding():
```

Comment:
Are these here to get pytest to see them? If so can you add a quick comment explaining that?

Reply:
yes, I think it can be used for test one operator like calling:  python -m pytest tests/align/align_test.py::test_add